client/src/components/ImageUploader/DropZone/DropZone.scss

```diff
@@ -0,0 +1,86 @@
+.Dropzone {
+  height: 160px;
+  width: 160px;
+  background-color: #fff;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  font-size: 16px;
+
+  .icon-wrapper {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    .icon {
+      opacity: 0.3;
+      height: 64px;
+      width: 64px;
+    }
+  }
+  
+  .FileInput {
+    display: none;
+  }
+
+  //circle dropZone
+  @keyframes lds-eclipse {
+    0% {
+      -webkit-transform: rotate(0deg);
+      transform: rotate(0deg);
+    }
+    50% {
+      -webkit-transform: rotate(180deg);
+      transform: rotate(180deg);
+    }
+    100% {
+      -webkit-transform: rotate(360deg);
+      transform: rotate(360deg);
+    }
+  }
+  @-webkit-keyframes lds-eclipse {
```

Comment:
Why 2 times with vendor prefix and without? Webpack should have postcss/autoprefixer, so with that we are ok with vendor prefixes. Could you recheck that pls? Or maybe I miss some point? If plugin is enabled, remove pls with vendor prefix